ui/src/services/PodUtils.js

```diff
@@ -0,0 +1,52 @@
+import { fromMilliSectoAge } from './utils.js';
+
+// Return the pod list for the Pod Tab in Node Page
+export const getPodsListData = (nodeName, pods) => {
+  const podsList = pods?.filter((pod) => pod.nodeName === nodeName);
+
+  return podsList?.map((pod) => {
+    let ageObject, age;
+    ageObject = fromMilliSectoAge(new Date() - pod.startTime);
+    // There are in total 10 cases to display the `Age` in two time unit maximum.
+    if (ageObject.day > 0 && ageObject.hour > 0) {
+      // start with `d`
+      age = ageObject.day + 'd' + ageObject.hour + 'h';
+    } else if (ageObject.day > 0 && ageObject.minute > 0) {
+      age = ageObject.day + 'd' + ageObject.minute + 'm';
+    } else if (ageObject.day > 0 && ageObject.second > 0) {
+      age = ageObject.day + 'd' + ageObject.second + 's';
+    } else if (ageObject.day > 0) {
+      age = ageObject.day + 'd';
+    } else if (ageObject.hour > 0 && ageObject.minute > 0) {
+      // start with `h`
+      age = ageObject.hour + 'd' + ageObject.minute + 'm';
+    } else if (ageObject.hour > 0 && ageObject.second > 0) {
+      age = ageObject.hour + 'd' + ageObject.second + 's';
+    } else if (ageObject.hour > 0) {
+      age = ageObject.hour + 'h';
+    } else if (ageObject.mintue > 0 && ageObject.second > 0) {
```

Comment:
```suggestion
    } else if (ageObject.minute > 0 && ageObject.second > 0) {
```

Reply:
Simplified this `if/else if` by Sylvain's suggestion. 
Extract all the logic inside `fromMilliSectoAge` and added a unit test for this function.